scarb/build.rs

```diff
@@ -70,6 +74,12 @@ fn download_core(rev: &str) {
     let out_dir = env::var("OUT_DIR").unwrap();
     let core_path = PathBuf::from_iter([&out_dir, &format!("core-{}", ident(rev))]);
 
+    if is_docs_rs() {
+        eprintln!("Docs.rs build detected. Skipping corelib download.");
+        println!("cargo:rustc-env=SCARB_CORE_PATH={}", core_path.display());
```

Comment:
Have you tested that this does indeed build? It looks strange for me that it works if you didn't create a directory. We're using `include_dir!` on this path.

Reply:
I've tested it locally by running `DOCS_RS=1 cargo build -vv`.